doc/tutorials/starship.rst

```diff
@@ -0,0 +1,99 @@
+.. _starship:
+
+Running Jobs with StarshipEngine
+################################
+
+.. sectionauthor:: Zeid Zabaneh <zeid@xanadu.ai>
+
+In this section, we provide a tutorial of the **StarshipEngine**, an engine used to connect to the Strawberry Fields platform and execute jobs remotely (e.g. on a quantum chip).
+
+Configuring StarshipEngine
+==========================
+
+Before using StarshipEngine, you need to configure the hostname and authentication token that will provide you access to the API. The easiest way is to create a configuration file named ``config.toml`` in your working directory. A typical file looks like this:
```

Comment:
This works now in local directory? I may have missed it, but didn't see where the code was changed to look for this locally

Reply:
By locally do you mean current working directory? It has always looked there first, but I forgot about that feature last week.

Reply:
Yep, that's what I meant. Could have saved a lot of hassle with different operating systems if I knew you could just put it there!

Reply:
Yes for sure. It is in the documentation as the first point, but it completely did not occur to me to use it then.